### ISSUE TITLE
Scripts failing to run with "No module named `window_commands`

### ISSUE THREAD

**Issue by SirGnip** [pvcraven/arcade#546]
Running a simple script with my arcade repo synced to 7111d49 is triggering ModuleNotFoundErrors:

```
$ cat test.py
import arcade
print(arcade.Sprite())

$ venv/Scripts/mypy test.py --strict
Success: no issues found in 1 source file

$ venv/Scripts/python test.py
Traceback (most recent call last):
  File "test.py", line 2, in <module>
    import arcade
  File "c:\repo\arcade\arcade\__init__.py", line 32, in <module>
    from window_commands import close_window
ModuleNotFoundError: No module named 'window_commands'
```

I did a quick tweak and tried adding `arcade.` to the beginning of the first `window_commands` imports in `__init__.py` to make it an absolute import path and reran `test.py`.  It got further.  Prefixing a `.` to make it an explicit relative import also allowed the script to get farther.  Python 3 no longer supports "implicit relative imports".

I also just tried running a couple scripts from the `examples/` directory in the arcade repo and got the same error.

**Comment by pvcraven** [comment]
Fixed. Maybe.

**Comment by pvcraven** [comment]
Fixed with 2.2.4.
